docs/source/index.md

```diff
@@ -130,18 +133,20 @@ https://github.com/elastic/{your-repository}/settings/pages
 
```

Comment:
There are two differences between this file and the readme:

1. MD Heading vs. Frontmatter heading.
2. The image path specified on L130 is different for each file.